integrated_channels/blackboard/client.py

```diff
@@ -732,3 +732,8 @@ def create_integration_content_for_course(self, bb_course_id, channel_metadata_i
             )
 
         return course_fully_created_response
+
+    def cleanup_duplicate_assignment_records(self, courses):
+        """
+        Not implemented yet.
+        """
```

Comment:
I'd add a log call here with a warning that it's not implemented for now.

Reply:
Log or raise a `NotImplemented` exception like in the base client?

Reply:
I'd say logger.error for now?